### PR TITLE
fix(merge-on-green)!: remove restrictions for checking reviews for merge on green

### PR DESCRIPTION
**Pull request by sofisl**
When we initially created merge-on-green, we weren't sure whether repos had set up branch protection adequately, so we wrote additional logic to make sure PRs had reviews. However, we've since instituted org-wide protection, this seems like an additional piece of logic now that causes a lot of confusion, see: https://github.com/googleapis/repo-automation-bots/issues/5193

### CODE REVIEW

File: packages/merge-on-green/test/merge-on-green.ts [hunk 1]
```diff
@@ -422,7 +422,7 @@ describe('merge-on-green wrapper logic', () => {
 
         const [reducedPRs] = handler.maybeReducePRList(prs);
 
-        assert.strictEqual(reducedPRs.length, 25);
+        assert.ok(reducedPRs.length === 24 || reducedPRs.length === 25);
```
Comment (chingor13) [inline packages/merge-on-green/test/merge-on-green.ts:425]: We should be confident in this value.

**Review by chingor13 — CHANGES_REQUESTED** [review]
Main change seems ok, but a nit about the test

File: packages/merge-on-green/test/merge-on-green.ts (hunk 1 above)
Comment (sofisl) [reply packages/merge-on-green/test/merge-on-green.ts:425]: I think it's OK if we're not confident. It's produced by a random value, and the only requirement is that it has to be under 25 (the chunk size): https://github.com/googleapis/repo-automation-bots/blob/ce561fe11471717eaa93aea7cdf2fe2922140b23/packages/merge-on-green/src/merge-on-green.ts#L87. I've updated the test to reflect that.

**Comment by sofisl** [comment]
Friendly ping @chingor13

**Comment by Iben1993** [comment]
> Amikor kezdetben az egyesületet--zöldre hoztuk létre, nem voltunk biztosak abban, hogy a repos megfelelően létrehozta-e a fióktelepek védelmét, ezért további logikát írtunk annak érdekében, hogy a PRok legyenek felülvizsgálatok. Azonban, azóta, hogy bevezettük az org-wide védelmet, ez úgy tűnik, mint egy további logika most, amely sok zűrzavart okoz, lásd: #5193

**Comment by Iben1993** [comment]
LICENSE

**Review by chingor13 — APPROVED** [review]

**Pull request merged**